drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -253,6 +253,46 @@ static int nrf5_set_txpower(struct device *dev, s16_t dbm)
 	return 0;
 }
 
+static int handle_ack(struct nrf5_802154_data *nrf5_radio)
+{
+	u8_t ack_len = nrf5_radio->ack_frame.psdu[0] - NRF5_FCS_LENGTH;
+	struct net_pkt *ack_pkt;
+	int err = 0;
+
+	ack_pkt = net_pkt_alloc_with_buffer(nrf5_radio->iface, ack_len,
+					    AF_UNSPEC, 0, K_NO_WAIT);
+	if (!ack_pkt) {
+		LOG_ERR("No free packet available.");
+		err = -ENOMEM;
+		goto free_nrf_ack;
+	}
+
+	if (net_pkt_write(ack_pkt, nrf5_radio->ack_frame.psdu + 1,
```

Comment:
For an outsider, it's of course not clear why you offset "raw" buffer by 1 here.

Reply:
Will add comment on that.

Reply:
@mike-scott isn't your request for change (typo) wasn't fixed ? Could you remove your -1 ?

Reply:
Thank you for the ping.  Approved.